From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Thu, 20 Sep 2018 17:48:59 -0700
Subject: resource_file_conflict.patch

Resolve conflict between //chrome's .pak files and //electron's. The paths
that chrome code hardcodes require that we generate resources at these
paths, but GN throws errors if there are multiple targets that generate the
same files.

diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index 4865a69fe984b428c4b6591d11d778e6c2669d2d..e8795d76f895580a398f748724b55eb5848a3528 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1638,6 +1638,11 @@ if (is_chrome_branded && !is_android) {
   }
 }
 
+if (is_electron_build) {
+  group("packed_resources") {
+    public_deps = [ "//electron:packed_resources" ]
+  }
+} else {
 chrome_paks("packed_resources") {
   if (is_mac) {
     output_dir = "$root_gen_dir/repack"
@@ -1659,6 +1664,7 @@ chrome_paks("packed_resources") {
     ]
   }
 }
+}
 
 repack("browser_tests_pak") {
   sources = [
-- 
2.16.2

